src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantsController.kt

```diff
@@ -0,0 +1,170 @@
+  @PostMapping
+  fun createParticipant(
+      @RequestBody payload: CreateParticipantRequestPayload
+  ): GetParticipantResponsePayload {
+    val model =
+        participantService.create(
+            ParticipantModel.create(cohortId = payload.cohortId, name = payload.name))
+
+    return makeGetResponse(model)
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @ApiResponse409("There are projects associated with the participant.")
+  @DeleteMapping("/{participantId}")
+  @Operation(summary = "Deletes a participant that has no projects.")
+  fun deleteParticipant(@PathVariable participantId: ParticipantId): SimpleSuccessResponsePayload {
+    participantStore.delete(participantId)
+
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{participantId}")
+  @Operation(summary = "Gets information about a participant and its assigned projects.")
+  fun getParticipant(@PathVariable participantId: ParticipantId): GetParticipantResponsePayload {
+    val model = participantStore.fetchOneById(participantId)
+
+    return makeGetResponse(model)
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @Operation(summary = "Updates a participant's information.")
+  @PutMapping("/{participantId}")
+  fun updateParticipant(
+      @PathVariable participantId: ParticipantId,
+      @RequestBody payload: UpdateParticipantRequestPayload
+  ): SimpleSuccessResponsePayload {
+    participantService.update(participantId, payload::applyTo)
+
+    return SimpleSuccessResponsePayload()
+  }
+
+  private fun makeGetResponse(model: ExistingParticipantModel): GetParticipantResponsePayload {
+    // The number of projects per participant should never exceed low single digits and will usually
+    // be 0 or 1, so fetching them one at a time shouldn't be a bottleneck. If we ever have hundreds
+    // of projects per participant, we can add a custom query to fetch them all at once.
+    val cohort = model.cohortId?.let { cohortStore.fetchOneById(it) }
+    val projects = model.projectIds.map { projectStore.fetchOneById(it) }
+    val organizationsById =
+        projects
+            .map { it.organizationId }
+            .distinct()
+            .map { organizationStore.fetchOneById(it) }
+            .associateBy { it.id }
+
+    val projectPayloads =
+        projects.map { project ->
+          val organization = organizationsById[project.organizationId]!!
+          ParticipantProjectPayload(organization.id, organization.name, project.id, project.name)
+        }
+
+    return GetParticipantResponsePayload(
+        ParticipantPayload(model.cohortId, cohort?.name, model.id, model.name, projectPayloads))
+  }
+}
+
+data class ParticipantProjectPayload(
+    val organizationId: OrganizationId,
+    val organizationName: String,
+    val projectId: ProjectId,
+    val projectName: String,
+)
+
+data class ParticipantPayload(
+    val cohortId: CohortId?,
+    val cohortName: String?,
+    val id: ParticipantId,
```

Comment:
can we add phase to this payload? I think everywhere in the front-end that uses cohort name, phase is also displayed.